py3status/modules/external_script.py

```diff
@@ -22,6 +23,7 @@
 
 Format placeholders:
     {output} output of script given by "script_path"
+    {count_lines} number of lines in the output
```

Comment:
I want to avoid `count_` prefix here. Use `{line}` just like `{path}`.

If one day, somebody want specific lines, then we can do `{line_3}` too without messing up the placeholders. Also, it is much shorter and clean.
* `[\?if=line&color=line {line} lines]`
* `[\?if=count_lines&color=count_lines {count_lines} lines]`.

Reply:
Why? In my mind that's more confusing, why `line_1` is a string and `line` is suddenly a number.

`num_lines` maybe?

Reply:
how about `line_count` then? no plurals, will stay next to `line_?` and yet explains well that this is a count.

Reply:
Similar things with percents... eg `{brightness}` instead of `{brightness_percent}`. Sometimes, we add `_percent` because of sysdata's `{mem_used}`.  `{mem_used}` is same as `{line}` since we don't suffix `{mem_used}` with anything.

We have to consider `format` too. If we prefix this with `_count`, then everything gets expanded much longer. Placeholder, if statements, color thresholds, etc. This approach where we don't be explicit with counts seems to work well for several modules I worked on... such as `mail` where I don't do `count_mail`. The users can use `[\?if=mail {mail} Mail]` instead of `[\?if=count_mail {count_mail} Mail]`.

Reply:
To be honest I don't see an issue with verbose configuration, at least it's readable without referring to docs. To a new user, it will not be obvious that `format = '{line}'` will actually print a number.

But, I am ready to give up on this and change to `line` if you really want so 🙂

Reply:
Please change. We need to get away with long names if we can... since most of the modules are small or simple enough not to warrant spelling out `_count`. Those simple module often are being used with threshold and other things too.

sysdata, pomodoro, taskwarrior, etc are not simple modules, but it looks like we can avoid them too... so I want to take advantage of that situation and extend this to all modules. It's a matter of verbosity level... I chalk it down a bit.